sql/wsrep_sst.cc

```diff
@@ -1297,8 +1301,6 @@ static void *sst_donor_thread(void *a) {
                 strerror(err));
   }
```

Comment:
Isn't it the security issue that after SST the user remains?

Reply:
The SST user is removed when the donor thread exits, I'm not sure why the other call was there. And even if it remains, it shouldn't be an issue, as the password isn't written down anywhere.